testsuite/tools/expect_test.ml

```diff
@@ -398,6 +398,7 @@ let usage = "Usage: expect_test <options> [script-file [arguments]]\n\
              options are:"
 
 let () =
+  Clflags.color := Some Never;
```

Comment:
I think that `Never` would be nicer written `Misc.Color.Never`, as I think this module is not opened in scope.